thredds_crawler/meta.yaml

```diff
@@ -19,6 +20,7 @@ requirements:
         - python
         - requests
         - lxml
+        - netCDF4
```

Comment:
Optional dependencies are up to the packager and I like to add them all.
So that is good. Thanks!

However, we can probably drop the `requests` and `lxml` at the build time.  But l will check that later, it depends if the code import itself to get the version.